v2/trace/oc/b3_dd_propagator.go

```diff
@@ -0,0 +1,151 @@
+// Package oc contains a propagation.HTTPFormat implementation for B3 and Datadog propagation.
+// The HTTPFormat is based on ochttp and ddtrace.
```

Comment:
Would like to have this file under v2/tracing/http/?

Reply:
maybe, but this is 100% for ochttp

Reply:
Three alternatives for folder structure:
v2/tracing/http/*.go
v2/tracing/http/oc/*.go
v2/tracing/oc/*.go
v2/tracing/oc/http/*.go

And you prefer the third option? I prefer either first or second one. I have no good arguments more than it look nice, it all depends on whats next will be added to the tracing-folder

Reply:
what about `v2/trace/ochttp/.go`?

Reply:
I'm good with the second one as well, I would like oc in the pkg

Reply:
I am good with that.